metriq_gym/benchmarks/__init__.py

```diff
@@ -1,23 +1,27 @@
 from metriq_gym.benchmarks.benchmark import Benchmark, BenchmarkData
 from metriq_gym.benchmarks.qml_kernel import QMLKernel, QMLKernelData
+from metriq_gym.benchmarks.clops import Clops, ClopsData
```

Comment:
[nit]: As we do with BSEQ and QML, since CLOPS is an acronym, do we want to write `Clops` as `CLOPS`?

Reply:
As I said in the review of the BSEQ PR, We shouldn't have name of Python classes that are all capitalized. `QMLKernel` is fine, `CLOPS` is not.